aws/resource_aws_ecs_service.go

```diff
@@ -101,7 +102,25 @@ func resourceAwsEcsService() *schema.Resource {
 				},
 				Set: resourceAwsEcsLoadBalancerHash,
 			},
-
+			"network_configuration": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"security_groups": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+						"subnets": {
+							Type:     schema.TypeList,
```

Comment:
As the ordering of subnets is not significant shouldn't this field be `TypeSet` to avoid potential spurious diffs in case the list of subnets comes back in a different order from the API?